lib/witchcraft/foldable.ex

```diff
@@ -30,7 +30,7 @@ defclass Witchcraft.Foldable do
   alias __MODULE__
   alias Witchcraft.{Apply, Ord, Monad, Monoid, Semigroup, Unit}
 
-  import Kernel, except: [length: 1, max: 2, min: 2]
+  import Kernel, except: [length: 1, max: 2, min: 2, then: 2]
```

Comment:
`Kernel.then/2` seems important like a pipe operator, so I think it's better not to except it. 
How do you think?